packages/adapter/src/types.ts

```diff
@@ -14,6 +14,6 @@ export enum AccountType {
 };
 
 export enum XcmInstructionNetworkType {
-  Null,
+  Null = 1,
```

Comment:
Does this mean the enums starting from 1?

Why not to use the default value 0?

Reply:
Avoid errors during conditional checks.
```
if (!network) throw new Error("network not set");
```

Reply:
No, this is a very basic javascript mistake. Please remember,

**DO NOT use `!network` in code**. It doesn’t know whether you want to check undefined, null or 0.

**Update all code like these**, and specify the condition check to either `!_.isUndefined(network)` or `!_.isNull(network)`.

Using `!network` shows that you have no idea how your code works. What value triggers the error message "network not set" exactly? You should not hack the enum value here because of some other code.

Reply:
There are so many `(!` in code!! 👎